src/utils.ts

```diff
@@ -0,0 +1,192 @@
+import { colours } from "../deps.ts";
+import Module from "./interfaces/module.ts";
+
+export function standardiseVersion(
+  importedVersion: string,
+  latestVersion: string,
+): string {
+  const importedVersionHasV = importedVersion.indexOf("v") === 0;
+  const latestVersionHasV = latestVersion.indexOf("v") === 0;
+
+  if (importedVersionHasV && !latestVersionHasV) {
+    latestVersion = "v" + latestVersion;
+    return latestVersion;
+  }
+
+  if (!importedVersionHasV && latestVersionHasV) {
+    latestVersion = latestVersion.substring(1);
+    return latestVersion;
+  }
+
+  return latestVersion;
+}
+
+interface DenoLandDatabase {
+  [key: string]: {
+    name: string;
+    repo: string;
+    desc: string;
+    owner: string;
+  };
+}
+/**
+ * @description
+ * Fetches Deno's `database.json` from the `deno_website2` GitHub repo
+ *
+ * @return {Promise<[key: string]: DenoLandDatabaseModule>} All 3rd party modules in Deno's registry
+ */
+async function getDenoLandDatabase(): Promise<DenoLandDatabase> {
+  const res = await fetch(
+    "https://raw.githubusercontent.com/denoland/deno_website2/master/database.json",
+  );
+  const denoDatabase: DenoLandDatabase = await res.json();
+  return denoDatabase;
+}
+export const denoLandDatabase: DenoLandDatabase = await getDenoLandDatabase();
+
+/**
+ * @description
+ * Fetches the latest std version
+ *
+ * @returns {Promise<string>} eg "0.57.0"
+ */
+async function getLatestStdRelease(): Promise<string> {
```

Comment:
on the note of services. you could have `DenoService.getLatestStdRelease()`

Reply:
> I think we should make some services for some of the logic. That's just my opinion. What I don't want to happen is we start putting stuff in `utils.ts` because it's easy.

Completely agree